content/blog/pinpoint-kubernetes/index.md

```diff
@@ -0,0 +1,203 @@
+---
```

Comment:
We need to change the URL to www.pulumi.com/blog/pinpoint (that is what is going in the Press Release that we sent to CNCF.